### PR TITLE
Updated es.yml file

### PR DESCRIPTION
**Pull request by natastro**
RE: Start a New Chapter translations needed #405

Pulled en.yml file from issue 405 
- kept any spanish translations from the es.yml document and moved them over
- translated the new lines that were found in en.yml

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-es:
+en:
```
Comment (jcn) [inline config/locales/es.yml]: This should not have changed from `es`

File: config/locales/es.yml [hunk 2]
```diff
@@ -92,10 +93,10 @@ es:
         header: Conoce a los Administradores
         about: Acerca de Nosotros
         news: "%{name} Noticias"
-        photo-alt-text: Foto de %{name}
+        photo-alt-text: "Foto de %{name}"
       apply-for-grant: Solicita una beca
       inactive-chapter: Este capítulo está inactivo.
-      inactive-chapter-reason: Un capítulo inactivo es uno que no ha otorgado una beca en seis meses. Si estás interesado en reiniciar este capítulo, por favor <a href="mailto:join@awesomefoundation.org">comunícate con nosotros</a>.
+      inactive-chapter-reason: Un capítulos inactivo es uno que no ha otorgado una beca en seis meses. Si estás interesado en reiniciar este capítulo, por favor <a href="mailto:join@awesomefoundation.org">comunícate con nosotros</a>.
```
Comment (jcn) [inline config/locales/es.yml]: Just confirming - is this correct as it is referring to a single chapter, not multiples.

File: config/locales/es.yml [hunk 3]
```diff
@@ -108,10 +109,20 @@ es:
       end-date: fecha final
       table:
         title: Título
-        id: Identificación
+        id: IIdentificaciónD
```
Comment (jcn) [inline config/locales/es.yml]: Not sure what happened here but this is wrong

File: config/locales/es.yml [hunk 4]
```diff
@@ -134,10 +146,11 @@ es:
       view-application: Véase la solicitud
       hiding-explanation: Esconder un proyecto esconderá sus detalles para todos; el título y la razón por la cual se escondió permanecerán visibles y puede revertirse fácilmente
       unhiding-explanation: Restaura este proyecto para que todos puedan verlo nuevamente
+
     new:
       title: ¿Estás listo para ser impresionante!?
-      description:  Simplemente completa la solicitud a continuación con información sobre tí y los detalles de tu proyecto.
-      apply: Solicita
+      description: Simplemente completa la solicitud a continuación con información sobre tí y los detalles de tu proyecto.
+      apply: AppSolicitaly
```
Comment (jcn) [inline config/locales/es.yml]: This looks like it got a bit mangled

File: config/locales/es.yml [hunk 5]
```diff
@@ -201,11 +216,49 @@ es:
...
+      photo_alt_2: Tres mujeres del capítulo de Amsterdam con un cartel leyendo "Ik Maak Leiden Awesomûrrr"
+      steps_headline: Cinco sencillos pasos
+      steps_intro: Poner en marcha un nuevo capítulo de Awesome Foundation es sencillo - nos gusta mantener las cosas simples.
+      existing_chapters_header: Mira los capítulos existentes
+      existing_chapters_text: Hay los capítulos de Awesome Foundation alrededor del mundo si es posible que ya hay un capítulo en su área. Mira el menú de "Capítulos" encima a ver si hay uno, y ponerse en contacto con ellos si ves uno cerca.
+      handbook_header: Descargar el manual
+      handbook_text: To help you get started, we collected the wisdom we've gained from giving out thousands of grants since 2009 and put it in our [New Chapter Handbook](http://wiki.awesomestudies.org/images/9/92/Awesome_Foundation_New_Chapter_Handbook.pdf). Para ayudarlo a comenzar, recogimos la sabiduría hemos ganado dando miles de subvenciones desde 2009 y ponerlo en nuestro [Nuevo manual de capítulos](http://wiki.awesomestudies.org/images/9/92/Awesome_Foundation_New_Chapter_Handbook.pdf).
```
Comment (jcn) [inline config/locales/es.yml]: This still has the english text in place

File: config/locales/es.yml [hunk 6]
```diff
@@ -278,14 +334,9 @@ es:
...
+              invalid: no es un sitio web válido
   meta:
     description: Promover los intereses de lo impresionante en el universo, $1,000 a la vez.
-  will_paginate:
-    previous_label: "&#8592; Anterior"
-    next_label: "Siguiente &#8594;"
-  helpers:
-    submit:
-      chapter:
-        update: Actualize el Capítulo
```
Comment (jcn) [inline config/locales/es.yml]: These translations should not have been removed.

File: config/locales/es.yml [hunk 7]
```diff
@@ -259,17 +316,16 @@ es:
...
     attributes:
       chapter:
         twitter_url: Twitter URL
         facebook_url: Facebook URL
         instagram_url: Instagram URL
         blog_url: Blog URL
-        email_address: Dirección de Email
-        hide_trustees: Ocultar administradores
-        country: País
-        time_zone: Zona horaria
```
Comment (jcn) [inline config/locales/es.yml]: These should not have been removed.

File: config/locales/es.yml [hunk 8]
```diff
@@ -108,10 +109,20 @@ es:
...
+      description: Usted recibirá un correo electrónico dentro de los próximos minutos. Contiene las instrucciones para cambiar su contraseña.
+    edit:
+      description: Su contraseña se ha reiniciado. Eliges una nueva contraseña abajo.
+      title: Cambiar su contraseña
+    new:
+      description: Recibir un enlace por correo electrónico a restablecer tu contraseña, por favor, introduzca su dirección de correo electrónico.
+      title: Restablecer su contraseña
   projects:
     wait-for-uploads-to-complete: Por favor espera a que tus imágenes terminen de subir antes de someter tu solicitud.
+
```
Comment (jcn) [inline config/locales/es.yml]: Looks like a stray newline was added

File: package-lock.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```
Comment (jcn) [inline package-lock.json]: Why was this added? This has nothing to do with the translation and should not be part of this commit.

**Comment by jcn** [comment]
@stroudn1 Hi! I am following up to see whether you have had a chance to review these requested changes. I would love to get your contributions merged into master.

**Comment by stroudn1** [comment]
> @stroudn1 Hi! I am following up to see whether you have had a chance to review these requested changes. I would love to get your contributions merged into master.

Hi @jcn - I apologize this is taking me so long. I haven't been on recently due to my mom having a stroke and she passed away Sunday. This week I'll be making arrangements but I'd like to take time to work on these changes this week or next if that's okay but I understand you need these done. Please let me know what you think. So sorry for the delay.

File: config/locales/es.yml (hunk 2 above)
Comment (stroudn1) [reply config/locales/es.yml]: I had that incorrect, sorry. I changed it to reflect capitulo

**Comment by stroudn1** [comment]
RE: Start a New Chapter translations needed #405

Changed en back to es
Changed 'capitulos' back to singular
Fixed mangled lines
Removed remaining english
Placed lines back that were removed
Removed stray line
Removed package-lock.json

I'm not sure what happened with some of these and I apologize with the huge delay but appreciate your patience. I'm also not sure what happened as far as the package-lock.json piece but tried removing it and submitting the changes. If you see anything else that needs fixed, please let me know - I'm happy to help. And thank you for the opportunity to let me submit PRs for this.

Have a good one!

**Comment by jcn** [comment]
@stroudn1 I'm so very sorry for your loss, and thanks so much for the update; no worries at all about the timing, obviously.

I just pushed a fix to the YAML file that was preventing it from being parsed properly and will be reviewing the translation as soon as possible.

**Review by jcn — APPROVED** [review]
